crates/codegen/runtime/generator/src/kinds/mod.rs

```diff
@@ -1,15 +1,52 @@
 use std::collections::BTreeSet;
+use std::rc::Rc;
 
+use codegen_ebnf::PlainWriter;
 use codegen_language_definition::model::{self, Identifier, Item, PredefinedLabel};
+use codegen_spec::SpecModel;
```

Comment:
I'm not sure I understand why does this need exposing `codegen_spec::SpecModel`, rather than just using the `codegen_ebnf::EbnfModel`?

The spec one is about building section/topic heirarchy, which is not needed here.

Reply:
Yeah you're right, it's unnecessary to use `SpecModel` here.